packages/grid/_modules_/grid/utils/material-ui-utils.ts

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { useEventCallback as muiUseEventCallback } from '@material-ui/core/utils';
 
 export function useEventCallback<T extends (...args: any[]) => any>(func: T): T {
-  // @ts-expect-error TODO remove wrapper once upgraded to v5
+  // @ts-ignore TODO remove wrapper once upgraded to v5
```

Comment:
Why did you change this line? `@ts-expect-error` was meant to break the CI once we update to v5, so we don't forget.

Reply:
I changed it while I was testing the v5. I can revert it but then every time we switch this needs to be removed.

Reply:
> every time we switch this needs to be removed.

When do we need to switch?

Reply:
While developing I guess. I was doing it quite often but then again I only did it because I was testing the 2 versions. Anyway, now that I think about it will be a problem only when you install and run locally v5 in the x repo which won't happen much often